### PR TITLE
Update event emitter methods

### PR DESCRIPTION
**Pull request by feross**
- Remove `off` which does not exist on `EventEmitter` or `process`. (This was my bad, as I added this in a PR a few years ago coming from my jQuery background hehe 😏)
- Add two new methods that are on `EventEmitter`.

### CODE REVIEW

File: browser.js [hunk 1]
```diff
@@ -164,10 +164,11 @@ function noop() {}
 process.on = noop;
 process.addListener = noop;
 process.once = noop;
-process.off = noop;
```
Comment (calvinmetcalf) [inline browser.js]: this could be argued to be a breaking change

**Comment by calvinmetcalf** [comment]
removing off could theoretically break stuff, otherwise 👍

**Comment by dadleyy** [comment]
are there plans to also add a [`listeners`](https://nodejs.org/api/events.html#events_emitter_listeners_eventname) function? [`jasmine@2.6`](https://github.com/jasmine/jasmine) was just released and [expects it to be defined](https://github.com/jasmine/jasmine/blob/master/src/core/GlobalErrors.js#L15) in node-like environments.

**Comment by calvinmetcalf** [comment]
@dadleyy good call

**Comment by calvinmetcalf** [comment]
@feross do you mind updating this if you're able, I can't add to this pull

**Comment by feross** [comment]
Ah, sorry I already deleted my fork.

What is the best way to add `listeners`? Should it always return an empty array? Or should we just make `process` actually inherit from `EventEmitter`?

**Comment by calvinmetcalf** [comment]
That's OK I'll just create a new one, empty array sounds right as
inheriting is probably a bad idea as it would blow up a lot of code that
used process

**Comment by feross** [comment]
Sounds good. Closing this PR then.

**Comment by calvinmetcalf** [comment]
done published etc

**Pull request closed without merging**